src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -450,6 +450,15 @@ def transform_request(self,
         if not filters:
             filters = {"file": {}}
         filters = filters['file']
+
+        for fltr in filters:
```

Comment:
Consider `facet`. Also, compared to `filter`, `fltr` saves two characters but is harder to read.